test/lib/next-webdriver.js

```diff
@@ -17,6 +22,22 @@ export default async function (appPort, pathname) {
       return browser
     } catch (ex) {
       console.warn(`> Error when loading browser with url: ${url}`)
+
+      // Try restarting chromedriver max twice
+      if (lc < 2) {
+        const chromedriver = require('chromedriver')
+        console.log('Trying to restart chromedriver with random port')
+        driverPort = await getPort()
+        chromedriver.stop()
+        chromedriver.start([`--port=${driverPort}`])
+        // https://github.com/giggio/node-chromedriver/issues/117
```

Comment:
Looking at https://github.com/giggio/node-chromedriver/pull/177 It seems like when you call it like

```js
await chromedriver.start([`--port=${driverPort}`], true)
```

You can eliminate the `wait-port` dependency.

Reply:
That's true, although it looks like the timeout is fixed that way and unfortunately I think we need to allow the timeout to be higher than 10 seconds. Thanks for looking

Reply:
ok, I filed https://github.com/giggio/node-chromedriver/issues/193